sdk/tables/Azure.Data.Tables/tests/TableClientQueryExpressionTests.cs

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml;
+using Azure.Core.TestFramework;
+using Azure.Data.Tables.Queryable;
+using NUnit.Framework;
+using static Azure.Data.Tables.Tests.TableServiceLiveTestsBase;
+
+namespace Azure.Data.Tables.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="TableServiceClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    public class TableClientQueryExpressionTests
+    {
+        private TableClient client { get; set; }
+        private const string TableName = "someTableName";
+        private const string Partition = "partition";
+        private const string Row = "row";
+        private const int SomeInt = 10;
+        private const double SomeDouble = 10.10;
+        private const long SomeInt64 = long.MaxValue;
+        private const string SomeString = "someString";
+        private const bool SomeTrueBool = true;
+        private const bool SomeFalseBool = false;
+        private readonly DateTime _someDateTime = DateTime.UtcNow;
+        private readonly DateTimeOffset _someDateTimeOffset = DateTimeOffset.UtcNow;
+        private readonly Guid _someGuid = Guid.NewGuid();
+        private readonly byte[] _someBinary = new byte[] { 0x01, 0x02, 0x03, 0x04, 0x05 };
+
+        [OneTimeSetUp]
+        public void TestSetup()
+        {
+            var service = new TableServiceClient(new Uri("https://example.com"));
+            client = service.GetTableClient(TableName);
+        }
+
+        [Test]
```

Comment:
Consider using test case source feature for these tests https://github.com/Azure/autorest.csharp/pull/791/files#diff-a3d0979b8d00c933aa081177dcfbf22bR13